### PR TITLE
fix(CI): tags do not trigger Flex builds

### PR DESCRIPTION
**Pull request by y3rsh**
## Overview

> The README.md changes describe the behavior of the changes
> <https://opentrons.atlassian.net/browse/RDEVOPS-132>

## ?

- Do we want pushes to main to still trigger Flex builds?

### CODE REVIEW

File: .github/workflows/start-ot3-build.yaml [hunk 1]
```diff
@@ -16,7 +14,7 @@ jobs:
   handle-push:
     runs-on: 'ubuntu-latest'
     if: github.event_name == 'push'
-    name: "Start an OT-3 build for a branch/tag push"
+    name: "Start an Flex build for a branch push of ${{ github.ref_name }}"
```
Comment (y3rsh) [inline .github/workflows/start-ot3-build.yaml:17]:
```suggestion
    name: "Start a Flex build for a branch push of ${{ github.ref_name }}"
```

File: .github/workflows/start-ot3-build.yaml [hunk 2]
```diff
@@ -39,8 +37,8 @@ jobs:
 
   handle-pr:
     runs-on: 'ubuntu-latest'
-    if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == 'Opentrons/ot3-firmware' && contains(github.event.pull_request.labels.*.name, 'ot3-build')
-    name: "Start an OT-3 build for a requested PR"
+    if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == 'Opentrons/ot3-firmware' && contains(github.event.pull_request.labels.*.name, 'flex-build')
+    name: "Start a Flex build for a requested PR"
```
Comment (y3rsh) [inline .github/workflows/start-ot3-build.yaml:41]:
```suggestion
    name: "Start a Flex build for a PR from branch ${{ github.ref_name }}"
```

**Review by sfoster1 — APPROVED** [review]
Looks good. We should keep pushes to `main` starting builds, yes, as long as they're not release builds. Makes it easier to test changes.

**Review by vegano1 — APPROVED** [review]
Thank you

**Pull request merged**